api/src/main/java/org/openmrs/module/m2sysbiometrics/client/M2SysV105Client.java

```diff
@@ -176,20 +179,27 @@ private M2SysCaptureResponse scanDoubleFingers() {
     }
 
     private FingerScanStatus checkIfFingerScanExists(M2SysCaptureResponse fingerScan) {
-        boolean existsLocally = false;
+        boolean existsLocally = isFingerScanExists(fingerScan, localBioServerClient);
         boolean existsNationally = false;
-        try {
-            existsLocally = localBioServerClient.isFingerScanExists(fingerScan);
-        } catch (APIException localClientAPIEx) {
-            LOG.error("Connection failure to local server.", localClientAPIEx);
-        }
 
-        try {
-            existsNationally = nationalBioServerClient.isFingerScanExists(fingerScan);
-        } catch (APIException nationalClientAPIEx) {
-            LOG.error("Connection failure to national server.", nationalClientAPIEx);
+        if (nationalBioServerClient.isServerUrlConfigured()) {
+            try {
+                existsNationally = isFingerScanExists(fingerScan, nationalBioServerClient);
+            } catch (WebServiceIOException wsException) {
```

Comment:
I would be on the safe side and just catch RuntimeException here

Reply:
ok I will correct it